dapp/src/components/NamePurchaseCard.tsx

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) 2025 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import clsx from 'clsx';
+
+interface NamePurchaseCardProps {
+    /**
+     * The name search
+     */
+    name: string;
+    /**
+     * Optional status message
+     */
+    supportingText?: string;
+    /**
+     * The price of the name
+     */
+    value?: string;
+    /**
+     * Currency symbol
+     */
+    currency?: string;
+    /**
+     * Supporting text of value
+     */
+    supportingTextValue?: string;
+    /**
+     *  Name is available
+     */
+    isAvailable: boolean;
+    /**
+     * Action buttons
+     */
+    children?: React.ReactNode;
+}
+
+export function NamePurchaseCard({
+    name,
+    supportingText,
+    value,
+    supportingTextValue,
+    isAvailable,
+    children,
+    currency,
+}: NamePurchaseCardProps): React.JSX.Element {
+    const bgCard = isAvailable ? 'bg-names-neutral-10' : 'bg-names-error-20';
+    const textColorStatus = isAvailable ? 'text-names-tertiary-80' : 'text-names-error-80';
+    const textStatus = isAvailable ? 'Available' : 'Unavailable';
+    const hasCurrency = currency ?? 'IOTA';
+
+    return (
+        <div
+            className={clsx(
+                'group relative w-full flex flex-col justify-between rounded-2xl p-[1px] space-y-4',
+                isAvailable && 'hover:bg-names-gradient-primary',
+            )}
+        >
+            <div
+                className={clsx(
+                    'group flex h-full w-full flex-col justify-between rounded-[calc(1rem-1px)] p-md--rs space-y-4',
```

Comment:
gap is 12px not 16px

```suggestion
                    'group flex h-full w-full flex-col justify-between rounded-[15px] p-md--rs space-y-4',
```